packages/notifi-react-card/lib/components/subscription/EventTypeCustomHealthCheckRow.tsx

```diff
@@ -299,6 +329,7 @@ export const EventTypeCustomHealthCheckRow: React.FC<
                 setErrorMessage('');
                 setSelectedIndex(null);
               }}
+              type="number"
               onBlur={handleCustomRatioButtonNewSubscription}
```

Comment:
should we prevent this handler from firing if `isNotificationLoading` is true as well?

Reply:
in fact, i would just `disable` this entire input if `isNotificationLoading` is true

Reply:
good point